libbpf-rs/src/map.rs

```diff
@@ -1066,6 +1060,55 @@ impl MapType {
     }
 }
 
+impl From<u32> for MapType {
+    fn from(value: u32) -> Self {
+        use MapType::*;
+
+        match value {
+            0 => Unspec,
+            1 => Hash,
+            2 => Array,
+            3 => ProgArray,
+            4 => PerfEventArray,
+            5 => PercpuHash,
+            6 => PercpuArray,
+            7 => StackTrace,
+            8 => CgroupArray,
+            9 => LruHash,
+            10 => LruPercpuHash,
+            11 => LpmTrie,
+            12 => ArrayOfMaps,
+            13 => HashOfMaps,
+            14 => Devmap,
+            15 => Sockmap,
+            16 => Cpumap,
+            17 => Xskmap,
+            18 => Sockhash,
+            19 => CgroupStorage,
+            20 => ReuseportSockarray,
+            21 => PercpuCgroupStorage,
+            22 => Queue,
+            23 => Stack,
+            24 => SkStorage,
+            25 => DevmapHash,
+            26 => StructOps,
+            27 => RingBuf,
+            28 => InodeStorage,
+            29 => TaskStorage,
+            30 => BloomFilter,
+            31 => UserRingBuf,
+            u32::MAX => Unknown,
```

Comment:
Can we remove this case? What purpose does it serve?

Reply:
consistency with the "old" implementation, but to be fair it's useless, so I removed it